packages/init/src/SliceMachineInitProcess.ts

```diff
@@ -965,6 +966,42 @@ ${chalk.cyan("?")} Your Prismic repository name`.replace("\n", ""),
 		]);
 	}
 
+	protected async setDefaultMasterLocale(): Promise<void> {
+		const documentsRead = await this.readDocuments();
+
+		if (documentsRead !== undefined && documentsRead.documents.length > 0) {
+			// if there are documents to push,
+			// we assume it's a starter which has a master locale already set
+			return;
+		}
+
+		return listrRun([
+			{
+				title: `Setting main content language...`,
+				task: async (_, task) => {
+					await this.manager.prismicRepository.setDefaultMasterLocale();
+					task.title = `Main content language set to ${chalk.cyan(
+						"English - United States",
+					)} 🇺🇸. You can change it anytime in your project settings.`;
+
+					try {
+						const { value: onboardingExperimentVariant } =
+							(await this.manager.telemetry.getExperimentVariant(
+								"shared-onboarding",
+							)) ?? {};
+						if (onboardingExperimentVariant === "with-shared-onboarding") {
```

Comment:
💡 Nit: Can you extract this to a function, since we already have it for `createNewRepository`, somethings like :

```ts
const isSharedOnboardingEnabled = await checkSharedOnboardingExperimentEnabled();
if (isSharedOnboardingEnabled) {
```

Reply:
good idea! will do

Reply:
done